tests/test-infra/clean_up.sh

```diff
@@ -15,6 +15,6 @@ for app in $installed_apps; do
 done
 
 echo "Trying to delete namespace..."
-kubectl delete namespace $1
+kubectl delete namespace $1 --timeout=10m
```

Comment:
with this change will it fail with error?

Reply:
yes I tested. It is failed. The hypothesis is that kubectl for win has the bug to check the status of ns deletion request. let's see how this fix goes.